deploy/olm-catalog/apicast-operator/apicast-operator.package.yaml

```diff
@@ -1,4 +1,6 @@
-packageName: apicast-operator-master
+packageName: apicast-community-operator
```

Comment:
I named this package "apicast-community-operator". In the possible future productized version it would be called "apicast-operator".
Both OperatorHub and RedHat community packages will be named the same (apicast-community-operator) as in the GitHub repository when those are contributed we are told to put the same content for both. This also implies that it's not possible to have both installed at the same time?

Reply:
Yes, but AFAIK you do not have operatorhub.io available in Openshift, so no problem.